dev/devicelab/lib/tasks/gallery.dart

```diff
@@ -359,7 +378,12 @@ class GalleryTransitionBuildTest extends BuildTestTask {
   @override
   String getApplicationBinaryPath() {
     if (applicationBinaryPath != null) {
-      return '${applicationBinaryPath!}/app-profile.apk';
+      if (deviceOperatingSystem == DeviceOperatingSystem.android) {
+        return '$applicationBinaryPath/app-profile.apk';
+      }
+      else if (deviceOperatingSystem == DeviceOperatingSystem.ios) {
+        return '$applicationBinaryPath/Flutter Gallery.app';
+      }
     }
 
     return 'build/app/outputs/flutter-apk/app-profile.apk';
```

Comment:
Should this also be dependent on the `DeviceOperatingSystem`?

Reply:
We currently support only android and ios. The return is the expected binary path, which already depends on the device os. Updated the logic a bit to make it clear.